docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Use project tests
+id: use-project-tests
+description: Custom SQL query tests
+sidebar_position: 8
+tags:
+  - how-to
+  - testing
+  - sql
+---
+
+Project tests are custom data tests that check for a passing condition, such as checking that the given query doesn’t return any unexpected rows. They are based on [dbt singular tests](https://docs.getdbt.com/docs/build/data-tests#singular-data-tests), but you don’t need to know dbt to use project tests in Prophecy. Project tests are useful for data users who know the important business questions but wouldn’t necessarily know how to write tests.
+
+Project tests perform a SQL query that checks the executed result of your project against a passing condition to see if it is successful. This is done by checking whether a generated table from your transformation meets a particular condition.
+
+Note: There could be many tests within a project. Each test is checking a table created by your choice of input Models, data sources, and transformation Gems.
+
+By default, and most often, the condition for a project test is simply a check that the given query does not return any failing rows.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/mvk1axfn7e?seo=false?videoFoam=true" title="Design a Pipeline Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Set up a test
+
+Since project tests share the same canvas as Models, you can set them up the same way you would develop and run a Pipeline.
+
+### Develop a test
+
+To develop a project test, start by opening a project:
+
+1. Under the Project Browser, click **Add Test**.
+
+   ![Add a new test](img/project-test-add-new.png)
+
+2. Enter your **Test Name**. Project tests are saved to a tests SQL file by default.
+
+3. Click **OK** to create your new test.
+
+4. Create your test Model by dragging and dropping Gems to your Visual canvas. Connect them to your Data Test. You can also write your test on the Code view. You can use as many Models and other database objects as you’d like in your test.
```

Comment:
I think it would be good to show this figure here: project-test-canvas.png

Reply:
SGTM. Since we're already using it in data-tests.md it'd be good to reuse.